### DISCUSSION TITLE
Invalid filter: 'naturaltime'

### DISCUSSION THREAD

**Discussion by romarioZ1000** [netbox-community/netbox#11230] [Q&A] [answered]
Good afternoon!
There is a problem on version 3.4.0 with page.
tell me which way to look to solve this problem?

.../extras/job-results/

When I click to view ID or time
![Screenshot_20221218_153352](https://user-images.githubusercontent.com/67051805/208303330-05143840-d07e-4e2e-b4f0-7c9e70a6cfcd.png)

this happens
![Screenshot_20221218_153424](https://user-images.githubusercontent.com/67051805/208303342-066238d8-97c1-4456-95df-6c6d476afae8.png)

Thank you!

**Comment by markkuleinio** [comment, marked as answer]
3.4.1 fixed this one: https://github.com/netbox-community/netbox/issues/11185

**Reply by romarioZ1000** [reply]
Thx
